test/common.bash

```diff
@@ -198,6 +198,9 @@ mkdir -p "$btnew"
 chmod 700 "$btnew"
 export BATS_TMPDIR=$btnew
 [[ $(stat -c %a "$BATS_TMPDIR") = '700' ]]
+# all temporary images all tagged TMPIMG so there is only 1 extra image at end
+# test case.
+export TMPIMG="foo"
```

Comment:
I think we don't need a variable here; the indirection doesn't buy us much (unless I am missing something?). I recommend you just call it `tmpimg` directly.